rosbag2_compression/src/rosbag2_compression/sequential_compression_writer.cpp

```diff
@@ -57,7 +57,9 @@ SequentialCompressionWriter::SequentialCompressionWriter(
 
 SequentialCompressionWriter::~SequentialCompressionWriter()
 {
-  close();
+  if (storage_) {
+    SequentialCompressionWriter::close();
+  }
```

Comment:
just curious that the same change needs to be applied to `SequentialCompressionWriter::open`? https://github.com/ros2/rosbag2/pull/1599/files#diff-39403f3879768a372efbcdd5a08b7d399259229334e96593ebf0ca16c2a76a89R228-R231 part is missing here?

Reply:
@fujitatomoya Added check in the commit [Return from writer's open() immediately if storage already open](https://github.com/ros2/rosbag2/pull/1635/commits/521de217665c7309c4cbf828ae08728131bbcfc5)